pack/Microsoft.NET.Sdk.Functions/Microsoft.NET.Sdk.Functions.csproj

```diff
@@ -104,9 +104,9 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net46'">
     <PackageReference Include="Newtonsoft.Json" Version="[9.0.1]" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="[2.2.0,2.3.0)" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="[2.2.0,2.3.0)" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="[1.1.0,1.2.0)" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="[2.2.0,2.4.0)" />
```

Comment:
Do we want to move the min version to 2.3.0? This will still use 2.2.0 unless user explicitly updates it.

Reply:
I'd be fine with that, but it means that we'd have to first release all the packages, before releasing this package and updating our tools to use this. @fabiocav ? How do you usually manage these release dependencies?

Reply:
I'll leave the range wider at this point - won't cause any problems.